src/python/native_common/ext_modules.py.in

```diff
@@ -71,75 +94,101 @@ def _create_module(module_name):
         os.path.join(abs_top_builddir, libprocess, '.libs', 'libprocess.a')
     ]
 
+    def cond_add_extra_objects(true_subst, false_subst, bundled, system):
+        """
+            Depending on the provided results of substitution by the configure
+            script, add either bundled or system ones to EXTRA_OBJECTS.
+            If adding bundled objects, check their existence
+            and fail if they are missing.
+        """
+        use_bundled = _get_ac_subst_bool(true_subst, false_subst)
+        if isinstance(bundled, basestring):
+            bundled = [bundled]
```

Comment:
I think if we want to support lists here, let's just *always* pass a list.

Reply:
I've split this method into two to address this issue.

An interface that usually accepts a list of ONE string is a perfect way for someone to shot himself in the leg when he forgets the square brackets: a single string is also iterable ;)